modules/contract/contracts.js

```diff
@@ -64,7 +64,7 @@ class Contracts {
     async getEntryPoints(args) {
         Logger.verbose(`Command : tezster extract-entry-points ${args}`);
         if (args.length < 1) {
-            Logger.warn('Incorrect usage - tezster list-entry-points <contract-absolute-path>');
+            Logger.warn('Incorrect usage - tezster extract-entry-points <contract-absolute-path>');
```

Comment:
why are you changing command name?

Reply:
the name is not being changed that list just got mistakenly typed insted of extract there.

Reply:
I think list-entry-points is better than extract.

Reply:
yeah will change it accordingly